cilium/README.md

```diff
@@ -114,24 +114,40 @@ Cilium contains two types of logs: `cilium-agent` and `cilium-operator`.
 
 For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying the parameters below.
 
-##### Metric collection
+Collecting logs is disabled by default in the Datadog Agent. To enable it, see [Kubernetes Log Collection][7].
+
+##### To collect `cilium-agent` metrics and logs: 
+
+- Metric collection
 
 | Parameter            | Value                                                      |
 |----------------------|------------------------------------------------------------|
-| `<INTEGRATION_NAME>` | `cilium`                                                   |
+| `<INTEGRATION_NAME>` | `"cilium"`                                                 |
 | `<INIT_CONFIG>`      | blank or `{}`                                              |
-| `<INSTANCE_CONFIG>`  | `{"agent_endpoint": "http://%%host%%:9090/metrics", "use_openmetrics": True}`       |
+| `<INSTANCE_CONFIG>`  | `{"agent_endpoint": "http://%%host%%:9090/metrics", "use_openmetrics": "true"}` |
 
-##### Log collection
-
-Collecting logs is disabled by default in the Datadog Agent. To enable it, see [Kubernetes Log Collection][7].
```

Comment:
I would still keep this line. The log collection is not enabled by default. Even with those params below if the logging is not enabled, it won't collect any.

Reply:
@steveny91 yes, I agree, hence why, instead of duplicating into both container agent and operator, I moved it upwards to: https://github.com/DataDog/integrations-core/pull/11656/files#diff-27abe71ae7c6058a11c4bc6558adb305760ae8a4f74ef8ee7003b4f3519b438fR117